src/crimson/osd/ops_executer.cc

```diff
@@ -364,6 +364,25 @@ OpsExecuter::watch_ierrorator::future<> OpsExecuter::do_op_notify(
   });
 }
 
+OpsExecuter::watch_ierrorator::future<> OpsExecuter::do_op_list_watchers(
```

Comment:
Technically it could be a `void`-returning method. Returning `future<>` just for the sake of visual consistency in `OpsExecuter::execute_op()`.